cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -699,9 +719,43 @@ pub mod bridging {
 			pub EthereumBridgeTable: sp_std::vec::Vec<NetworkExportTableItem> = sp_std::vec::Vec::new().into_iter()
 				.chain(BridgeTable::get())
 				.collect();
+
+			pub EthereumBridgeTableV2: sp_std::vec::Vec<NetworkExportTableItem> = sp_std::vec::Vec::new().into_iter()
+				.chain(BridgeTableV2::get())
+				.collect();
+		}
+
+		pub struct XcmForSnowbridgeV2;
+
+		impl Contains<Xcm<()>> for XcmForSnowbridgeV2 {
+			fn contains(xcm: &Xcm<()>) -> bool {
+				let mut instructions = xcm.clone().0;
+				let result = instructions.matcher().match_next_inst_while(
+					|_| true,
+					|inst| {
+						return match inst {
+							AliasOrigin(..) => Err(ProcessMessageError::Unsupported),
+							_ => Ok(ControlFlow::Continue(())),
+						}
+					},
+				);
+				result.is_err()
+			}
 		}
 
-		pub type EthereumNetworkExportTable = xcm_builder::NetworkExportTable<EthereumBridgeTable>;
+		pub struct XcmForSnowbridgeV1;
+
+		impl Contains<Xcm<()>> for XcmForSnowbridgeV1 {
+			fn contains(_: &Xcm<()>) -> bool {
+				true
```

Comment:
Shouldn't this be better like "does not contain AliasOrigin" instead of just "true"?
Or maybe something like:
```
fn contains(xcm: &Xcm<()>) -> bool {
    !XcmForSnowbridgeV2::contains(xcm)
}
```

or even easier, you dont need `XcmForSnowbridgeV1` at all, maybe just use `EverythingBut`:
```
		pub type EthereumNetworkExportTable =
			xcm_builder::NetworkWithXcmExportTable<EthereumBridgeTable, EverythingBut<XcmForSnowbridgeV2>>;
```


Anyway, neither XcmForSnowbridgeV2 nor XcmForSnowbridgeV1 is aligned with description:
```
The predicate for V1 XCM will look for

WithdrawAsset(...) or ReserveAssetDeposited # For erc20 and PNA respectively
ClearOrigin
BuyExecution

The predicate for V2 XCM will look for

WithdrawAsset or ReserveAssetDeposited or TeleportedAssetRevceived or * 
PayFees(fees)
ALiasOrigin
```

Reply:
Refactor with EverythingBut in https://github.com/paritytech/polkadot-sdk/pull/6074/commits/8cb272e80c395a55feb3f5da77af4bf085bf09c5

> neither XcmForSnowbridgeV2 nor XcmForSnowbridgeV1 is aligned with description
On AH we just inspect xcm which contains a specific instruction, we will validate the full xcm in [Exporter](https://github.com/paritytech/polkadot-sdk/blob/b649f4a4695b5444e73ef755658cf5c6efdc68b6/bridges/snowbridge/primitives/router/src/outbound/mod.rs#L121) on BH.